### DISCUSSION TITLE
Unable to build helidon mp as uber jar

### DISCUSSION THREAD

**Discussion by hungchu0912** [helidon-io/helidon#8196] [Q&A] [answered]
Is it possible to build Helidon MP as Uber jar? In some circumstances, we need an Uber jar only for convenience and compatibility with the current pipeline structuring. I have tried with maven-shade-plugin plugin but it doesn't work out.

**Comment by romain-grecourt** [comment, marked as answer]
Far/Uber jars are not support by Helidon ; see https://medium.com/helidon/evil-of-fatjar-3c3011b4bd55
